coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -1514,4 +1514,14 @@ def forward(self, x):
             expected_results=expected_results,
             input_as_shape=False,
             backend=backend,
-        )
\ No newline at end of file
+        )
+        
+class TestConstantPad:
```

Comment:
Can you please update the PR by removing the translation method for `constant_pad_nd` since that's already in master, but adding this test case to the constant pad test class, here: https://github.com/apple/coremltools/blob/6245e289187c95d32b6ff65bc6348310ebbdf71a/coremltools/converters/mil/frontend/torch/test/test_torch_ops.py#L1367

Then we can merge this change!